### ISSUE TITLE
Support CLion

### ISSUE THREAD

**Issue by zcourts** [Atsky/haskell-idea-plugin#101]
Would it need more than
```xml
    <idea-version since-build="139.1659" until-build="142.9999"/>
```
In https://github.com/Atsky/haskell-idea-plugin/blob/master/plugin/META-INF/plugin.xml#L8 to enable other JetBrains platforms to install this plugin?

I'm using CLion for C/C++ and it'd be handy to be able to edit the Haskell bits in there as well.

**Comment by Atsky** [comment]
To make plugin installable on CLion you shoiuld remove line.
```XML
<depends>com.intellij.modules.platform</depends>
```

But I think plugin won't work properly, and will crush. CLion has slighly different API.
I support IntelliJ IDEA at first place. Maybe I will add CLion support later.

**Comment by zcourts** [comment]
I'll give it a try and see what breaks. What version of the Kotlin plugin is required to build? I have several Kotlin syntax errors at the moment.
![screen shot 2015-06-24 at 02 44 59](https://cloud.githubusercontent.com/assets/692396/8321135/1ee09680-1a1b-11e5-9858-bb0d5e7d849a.png)

**Comment by Atsky** [comment]
I use last version of plugin (0.12.200). I think you need last version of IDEA to install last Kotlin version. Also you need JDK 1.6 to compile plugin.

**Comment by zcourts** [comment]
Yup that was my bad, created Intellij SDK with 1.8 and then 1.7 twice instead of with 1.6...built and installed. Seems to work so far.

I'll try to fix anything that comes up and report back but spent some time using it in CLion last night and had no issues so far.

**Comment by KitsuneRal** [comment]
I'm not sure how much active this is but I'll just give a note about the current status with CLion on Windows:
1. Trying to install ghc-mod from inside CLion eventually runs into failure to automatically configure old-time. Solved by manually running `cabal install` from the (MSYS2) command line.
2. SDK is not found, and I don't know a way of specifying its location, since CLion has hardcoded the toolchain for C++, and there's no IDEA "Project SDK" setting.
That's pretty much how far I could go.
